### PR TITLE
GTKAda 21.0.0

### PR DESCRIPTION
**Pull request by Fabien-Chouteau**

### CODE REVIEW

File: index/gt/gtkada/gtkada-21.0.0.toml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+description = "Ada Web Server"
```
Comment (Jellix) [inline index/gt/gtkada/gtkada-21.0.0.toml]: Should read "GtkAda" or similar, not "Ada Web Server".

File: index/gt/gtkada/gtkada-21.0.0.toml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+website = "https://github.com/adacore/gtkada"
+name = "gtkada"
+version = "999.0.0"
```
Comment (Jellix) [inline index/gt/gtkada/gtkada-21.0.0.toml]: Tad bit on the high side. "21.0.0".

File: index/gt/gtkada/gtkada-21.0.0.toml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+authors = ["AdaCore"]
+licenses = ["GMGPL 3.0"]
+maintainers = ["chouteau@adacore.com"]
+maintainers-logins = ["Fabien-Chouteau"]
+project-files = ["install_dir/lib/gnat/gtkada.gpr"]
+tags = ["aws", "web", "server", "http", "https", "soap", "wsdl", "ssl"]
```
Comment (Jellix) [inline index/gt/gtkada/gtkada-21.0.0.toml]: Not exactly the tags I expect for a UI library.

**Review by mosteo — APPROVED** [review]

**Comment by mosteo** [comment]
Our first conditional dependency, I'd say.

**Pull request merged**